brave/brave-core
# Update name of VisualElementsManifest (used by Windows)

Fixes https://github.com/brave/brave-browser/issues/5410

Implemented as a patch because it's very unlikely to be changed (last changed 2015 w/ https://chromium.googlesource.com/chromium/src/+/63a8d3b86b0f45d7c99c85ae57f45ba208254680)

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Install on Windows
2. Pin Brave to Start menu
3. Verify gray background is shown for tile

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
